modules/update-duration-example.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/understanding-openshift-update-duration.adoc
+
+:_content-type: CONCEPT
+[id="update-duration-example_{context}"]
+= Example update duration of cluster Operators
+
+image::update-duration.png[A diagram displaying the periods during which cluster Operators update themselves during an OCP update]
+
+The previous diagram shows an example of the time that cluster Operators might take to update to their new versions.
+The example is based on a three-node AWS OVN cluster, which has a healthy compute `MachineConfigPool` and no workloads that take long to drain, updating from 4.13.0 to 4.14.0.
+
+[NOTE]
+====
+The specific update duration of a cluster and its Operators can vary based on several cluster characteristics, such as the target version, the amount of nodes, and the types of workloads scheduled to the nodes.
+====
+
+Each cluster Operator has characteristics that affect the time it takes to update itself.
+For instance, the Kube API Server Operator in this example took over eleven minutes to update because it was serving customer requests and is designed to be conservative about draining out existing requests before shutting down a pod.
+In the case of this Operator, update speed is sacrificed to help prevent and limit disruptions to cluster functionality during an update.
```

Comment:
I could go either way in terms of placement, both options have their merits.

Reply:
Alright, moving this module to be under the two general discussion sections as I don't have a strong preference to keep it where it is now.

Reply:
the Kube API Server Operator in this example took over eleven minutes to update because kube-apiserver provides graceful termination support. For existing requests in-flight are allowed to complete gracefully. This may result in a longer shutdown of the kube-apiserver.